usr.sbin/moused/moused.c

```diff
@@ -1566,7 +1566,7 @@ r_name(int type)
     const char *unknown = "unknown";
 
     return (type == MOUSE_PROTO_UNKNOWN ||
-	type >= (int)(sizeof(rnames) / sizeof(rnames[0])) ?
+	type >= (int)nitems(rnames) ?
```

Comment:
@bsdimp should this be flattened into 2 lines instead of 3? Namely:
```
    return (type == MOUSE_PROTO_UNKNOWN ||
        type >= (int)nitems(rnames) ? unknown : rnames[type]);
```

Reply:
Yes, I think so...